src/x11/x.c

```diff
@@ -161,8 +161,24 @@ static bool x_win_composited(struct window_x11 *win)
 void x_display_surface(cairo_surface_t *srf, window winptr, const struct dimensions *dim)
 {
```

Comment:
Just a note here. dim->x and dim->y aren't used anywhere anymore. This is just temporary though, because after #855 the window position code will be shared again.

Reply:
I was going to remove it from struct, but some other parts of the code are still using it.